frontend-react/src/App.tsx

```diff
@@ -140,6 +140,11 @@ const App = () => {
                                 {/* TODO: AuthorizedRoute needs to take many potential auth groups.
                                  *  We should fix this when we refactor our permissions layer.
                                  */}
+                                <AuthorizedRoute
+                                    path="/submissions/:actionId"
+                                    authorize={PERMISSIONS.PRIME_ADMIN}
+                                    component={SubmissionDetails}
+                                />
```

Comment:
Yeah, I was gonna expose it in another PR for senders, but figured I'd keep this work to just the submission details and corresponding functionality

Reply:
For senders, does this require adding an additional `AuthorizedRoute`?

Reply:
The work I'm doing today on a separate branch is going to expose that `AuthorizedRoute` and the `Submission` nav item to Senders :slightly_smiling_face: luckily no crossover with this PR, so I figured I'd keep them separate in case of an untimely decision to not show to Senders (which happened last time)